src/wallet/db.h

```diff
@@ -28,6 +28,7 @@ class CDBEnv
     bool fDbEnvInit;
     bool fMockDb;
     boost::filesystem::path path;
```

Comment:
Do add a comment then, on why the duplicated field was added. Otherwise people will be really confused to read this code.